apps/www/pages/install.js

```diff
@@ -0,0 +1,62 @@
+import Frame from '../components/Frame'
+import StatusError from '../components/StatusError'
+
+import { createGetServerSideProps } from '../lib/apollo/helpers'
+import {
+  matchAndroidUserAgent,
+  matchIOSUserAgent,
+} from '../lib/context/UserAgentContext'
+import { getNativeAppVersion } from '../lib/withInNativeApp'
+
+const InstallNativeAppPage = ({ clientRedirection }) => (
+  <Frame raw>
+    <StatusError status={404} clientRedirection={clientRedirection} />
+  </Frame>
+)
+
+export default InstallNativeAppPage
+
+const getQueryString = (req) => {
```

Comment:
why not use `import { ParsedUrlQuery } from 'querystring'` or URLSearchParams API ?

Reply:
Would have to use something like:
```
import {format,parse} from url
format({
  pathname: '/',
  query: parse(req.url).query
})
format({
  pathname: '/probelesen',
  query: parse(req.url).query
})
```

But no real reason, just felt like it was easy and predictable enough to do with simple string operations.

Reply:
got it